leakcanary-android-core/src/main/java/leakcanary/internal/tv/TvToast.kt

```diff
@@ -25,10 +29,16 @@ internal object TvToast {
     text: CharSequence
   ): Toast {
     val toast: Toast = Toast.makeText(activity, text, Toast.LENGTH_LONG)
-    val textView = toast.view.findViewById<TextView>(android.R.id.message)
-    textView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.leak_canary_icon, 0, 0, 0)
-    textView.compoundDrawablePadding =
-      activity.resources.getDimensionPixelSize(R.dimen.leak_canary_toast_icon_tv_padding)
+
+    // Custom toast views are deprecated from API level 30 on, see
+    // https://developer.android.com/reference/android/widget/Toast#getView()
+    if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.Q) {
+      val textView = toast.view.findViewById<TextView>(android.R.id.message)
+      textView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.leak_canary_icon, 0, 0, 0)
+      textView.compoundDrawablePadding =
+        activity.resources.getDimensionPixelSize(R.dimen.leak_canary_toast_icon_tv_padding)
```

Comment:
I wonder why this would be a problem on TV but not on normal Android, which currently shows the custom toast just fine.

Reply:
Looking into it more, it seems like the key issue wasn't just showing a custom toast, but instead showing a custom toast when the app is in the background. In the foreground, it shouldn't throw.

I wonder what triggered this crash on TV.